bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/ComponentUtils.java

```diff
@@ -61,19 +62,19 @@ public static String getEncodedPath(@NotNull String path) {
 
     /**
      * Checks if the given resource if a core adaptive form container
-     * 
+     *
      * @param resource reference to the {@link Resource}
      * @return true, if it is an adaptive form container, false otherwise
      */
     @NotNull
     public static boolean isAFContainer(@NotNull Resource resource) {
         String fieldType = resource.getValueMap().get("fieldType", String.class);
-        return FORM_FIELD_TYPE.equals(fieldType);
+        return FORM_FIELD_TYPE.equals(fieldType) || FRAGMENT_FIELD_TYPE.equals(fieldType);
```

Comment:
Can we think of some other way, this is causing a confusion ? Looks like we are are supporting a new field type fragment in the headless specification, but that is not the case

Reply:
not needed any longer as this was part of static implementation.